### PR TITLE
Avatar gravatar

### PR DESCRIPTION
**Pull request by megankbarker**
Set default avatar in navbar to gravatar mystery man as a placeholder until we do the Amazon s3 credentials for our avatar image uploads.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class UsersController < ApplicationController
+  
+
+  def new
```
Comment (woodbrettm) [inline app/controllers/users_controller.rb:3]: def new and def edit are handled by devise :). You'll only need `def show` in this controller.

File: app/views/application/_main_sidenav.html.erb [hunk 2]
```diff
@@ -8,7 +8,7 @@
       <% end %>
     </li>
     <li>
-      <%= link_to edit_user_registration_path do %>
```
Comment (woodbrettm) [inline app/views/application/_main_sidenav.html.erb:11]: Should still be edit_user_registration_path.

File: config/routes.rb [hunk 3]
```diff
@@ -6,6 +6,7 @@
   
   devise_for :users, controllers: {sessions: 'users/sessions', registrations: 'users/registrations'}
   resources :games
+  resources :users, only: [:show, :edit]
```
Comment (woodbrettm) [inline config/routes.rb]: only show

File: app/views/application/_main_sidenav.html.erb (hunk 2 above)
Comment (megankbarker) [inline app/views/application/_main_sidenav.html.erb:11]: done

File: config/routes.rb (hunk 3 above)
Comment (megankbarker) [inline config/routes.rb]: done

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (megankbarker) [inline app/controllers/users_controller.rb:3]: done

**Comment by megankbarker** [comment]
-We've got gravatar running with a default user image of mystery man (when users are signed in, but do not have gravatar accounts associated with their email). 
-In order for a gravatar avatar to be displayed, a user's email address must be associated with an established gravatar/wordpress account.  
-I left the avatar model in the app in case we did decide to add avatar image uploading capabilities at some point.

**Pull request merged**